### PR TITLE
Started workshops from scratch

### PR DESCRIPTION
**Pull request by tjhasan**

### CODE REVIEW

File: applications/migrations/0028_auto_20191214_2351.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```
Comment (Andros415) [inline applications/migrations/0028_auto_20191214_2351.py]: The two migrations to applications may messed up how it's setup on the server side

Comment (Andros415) [reply applications/migrations/0028_auto_20191214_2351.py]: Everything else looks good though :)

Comment (tjhasan) [reply applications/migrations/0028_auto_20191214_2351.py]: It should still be okay on the server rn right? Since it hasn't been merged yet

Comment (Andros415) [reply applications/migrations/0028_auto_20191214_2351.py]: Yeah it should still be ok but we should delete the application migrations before merging the workshop changes

**Review by Andros415 — APPROVED** [review]
Looks good otherwise

**Pull request merged**